index.html

```diff
@@ -1,15 +1,158 @@
+					<li class="fb"><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+					<li class="tw"><a href="#"><i class="fab fa-twitter"></i></a></li>
+					<li><a href="#" >Language</a></li>
+					<li><a href="#" >Login</a></li>
+					<li><a href="#" >Register</a></li>
+				</ul>
+			</nav>
+		</div>
+		<div class="nav2">
+			<nav class="nav2a container">
+				<a href="" class="logo"><img src="images/logo_wtsq_1.png" alt="logo header"></a>
+				<ul class="ul-2">
+					<li><a href="about.html" >About</a></li>
+					<li><a href="#" >Program</a></li>
+					<li><a href="#" >Participants</a></li>
+					<li><a href="#">Contact</a></li>
+					<li><a href="#">WT Campaign</a></li>
+				</ul>
+			</nav>
+		</div>
+	</header>
+<!-- Hero SECTION -->
+	<section id="headline">
+		<div class="inner-hero container">
+			<div class="hero-desktop">
+				<blockquote>&ldquo; Programming & Development&rdquo;</blockquote>
+				<h1 class="hero-title">WEB TECHS SOFTWARE QUALITY SUMMIT 2022</h1>
+				<p class="hero-paragraph">One of the biggest conventions on software quality invites to Madrid for the fifth time now. One track will be focused on the key subject exclusively.</p>
+				<h3 class="hero-date">2022.01.06(THU)~07(FRI)</h3>
+				<h5 class="hero-join">@ Central Conferance hall of Marid and more</h5>
+			</div>
+		</div>
+	</section>
+<!-- PROGRAMME SECTIONS-->
+	<section id="courses">
+		<div class="inner-course container">
+			<h2 class="course-menu">Main Programe</h2>
+			<hr/>
+			<ul class="ul-courses">
+				<li class="course-item">
+					<i class="fas fa-sitemap"></i>
+					<h4>Architecture</h4>
+					<p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Delectus ad placeat temporibus repudiandae.</p>
+				</li>
+				<li class="course-item">
+					<i class="fas fa-users"></i>
+					<h4>Usabilty</h4>
+					<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Illo placeat quis asperiores sed.</p>
+				</li>
+				<li class="course-item">
+					<i class="fas fa-stethoscope"></i>
+					<h4>Testing</h4>
+					<p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Aut ipsum labore necessitatibus incidunt!</p>
+				</li>
+				<li class="course-item">
+					<i class="fas fa-user-lock"></i>
+					<h4>Security</h4>
+					<p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil nostrum cumque incidunt debitis.</p>
+				</li>
+				<li class="course-item">
+					<i class="fas fa-step-forward"></i>
+					<h4>Procedures</h4>
+					<p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Itaque quidem harum repellendus beatae.</p>
+				</li>
+			</ul>
+			<h3 class="more-courses">Join WTSQ summit 2022</h3>
+			<div class="div-program">
+				<h3 class="hole-program">SEE THE HOLE PROGRAM</h3>
+			</div>
+		</div>
+	</section>
+<!-- FEATURED SPEAKERS SECTIONS-->
+	<section id="intructors">
+		<div class="inner-istructors container">
+			<h3 class="instructors-title">Fetaured Speakers</h3>
```

Comment:
- [Optional] There is a typo here, the project would look better if you fixed it